src/docs/docker-plugin.md

```diff
@@ -59,13 +71,22 @@ Also during volume creation, customer is able to set additional options for new
 Example:
 
 ```
-docker volume create -d nexenta/nexentaedge-nfs-plugin:dev -o enableErasure=true -o erasureMode="4:2:rs" nfs01@clu1/ten1/buc1
+docker volume create -d nexenta/nexentaedge-nfs-plugin:dev -o enableErasure=true -o erasureMode="4:2:rs" ten1/buc1
 ```
 
 ### Run container with already created volume
 
 ```
-docker run -v nfs01@clu1/ten1/buc1:/Data -it ubuntu /bin/bash
+docker run -v nfs01@clu1/ten1/buc1:/Data -it ubuntu /bin/bash  (Full volume path specified)
+docker run -v ten1/buc1:/Data -it ubuntu /bin/bash  (Common volume path specified)
+```
+
+### Run container with non existent volume
+Docker will check volume existance, and if it not exists, new volume will be created
```

Comment:
Docker will check volume existence, and if it not exists, new volume will be created
Docker checks if volume exists, if it does not, then a new volume is created.